README.md

```diff
@@ -1 +1 @@
-# project-time-series-forecasting
\ No newline at end of file
+# project-time-series-forecasting
```

Comment:
Add a link to the project on Neptune.
Use https://github.com/neptune-ai/project-nlp#readme as a reference